protocol.h

```diff
@@ -15,10 +15,22 @@
 #include <string>
 #include "uint256.h"
 
-extern bool fTestNet;
-static inline unsigned short GetDefaultPort(const bool testnet = fTestNet)
+enum
```

Comment:
Give this enum a descriptive type name, and use the type name instead of `uint64_t` ?

Reply:
Done